### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by jimis**

### CODE REVIEW

**Comment by jimis** [comment]
I manually excluded the masterfiles changes since we have a separate changelog there now (we have to updated `changelog-generator` for that).

@kacf @estenberg @nickanderson please review

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,24 @@
+3.10.1:
+	New features/additions:
+	- Add: Ability to set default_repository via augments
```
Comment (nickanderson) [inline ChangeLog]: This default_repo thing is MPF. It could be redacted here.

**Comment by nickanderson** [comment]
One small comment, but other than that looks good.

**Comment by jimis** [comment]
ok, redacting that line and pushing again.

**Comment by olehermanse** [comment]
@jimis @nickanderson `sys.flavour` or `sys.flavor`, changelog says flavour, I think it's flavor, which is correct? Are we british or american?

**Comment by jimis** [comment]
On Wed, Mar 8, 2017 at 6:14 PM, Ole Herman Schumacher Elgesem <
notifications@github.com> wrote:

> @jimis <https://github.com/jimis> @nickanderson
> <https://github.com/nickanderson> sys.flavour or sys.flavor, changelog
> says flavour, I think it's flavor, which is correct? Are we british or
> american?
>
> @olehermanse we are both! Relevant code:

```
libenv/sysinfo.c-static void SetFlavour(EvalContext *ctx, const char
*flavour)
libenv/sysinfo.c-{
libenv/sysinfo.c:    EvalContextClassPutHard(ctx, flavour,
"inventory,attribute_name=none,source=agent,derived-from=sys.flavor");
libenv/sysinfo.c-    EvalContextVariablePutSpecial(ctx, SPECIAL_SCOPE_SYS,
"flavour", flavour, CF_DATA_TYPE_STRING, "source=agent");
libenv/sysinfo.c:    EvalContextVariablePutSpecial(ctx, SPECIAL_SCOPE_SYS,
"flavor", flavour, CF_DATA_TYPE_STRING,
"inventory,source=agent,attribute_name=none");
libenv/sysinfo.c-}
```

**Comment by olehermanse** [comment]
@jimis @nickanderson Because of the order of things, might be relevant to test that this works with the new CoreOS support. Check that coreos has both `sys.flavor` and `sys.flavour`.

**Pull request merged**